api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupSMCRegister.java

```diff
@@ -128,27 +141,51 @@ public String getVersion() {
         return l;
     }
 
+    private CohortDefinition getSmcUsedEncounters(){
+        CompositionCohortDefinition cd = new CompositionCohortDefinition();
+        cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+        cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+        cd.addSearch("client", ReportUtils.map(Cohorts.genderAndHasAncEncounter(false, true, "244da86d-f80e-48fe-aba9-067f241905ee"), "startDate=${startDate},endDate=${endDate}"));
```

Comment:
@ningosi 

1. Please rename the method as it will create issues 

2. You do not need the followup cohort as that is a subset of the client cohort